src/splinterdb_tests_private.h

```diff
@@ -20,26 +24,20 @@ splinterdb_cache_flush(const splinterdb *kvs);
 platform_heap_id
 splinterdb_get_heap_id(const splinterdb *kvs);
 
-const void *
+const task_system *
 splinterdb_get_task_system_handle(const splinterdb *kvs);
 
-const void *
+const platform_io_handle *
 splinterdb_get_io_handle(const splinterdb *kvs);
 
-const void *
+const rc_allocator *
```

Comment:
Actually, can you change these to return `cache *` and `allocator *` (i.e. the abstract types rather than the concrete implementations)?  Thank you!

Reply:
Done in final round of changes. 

Fyi; @rtjohnso -- Final CI job is running ... Once it passes, I will push this to `/main`.